PokeDemo/Screens/PokemonList/Model/Service.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Service.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+import Alamofire
+
+protocol ServiceRepresentable {
+    func sendRequestWithJSON(
+        endpoint: String,
+        method: HTTPMethod,
+        parameters: [String:Any]?,
```

Comment:
Can it be [String: String] instead of [String: Any]?

Reply:
Covered in 8c9007acbdcba689302ff066340d4f232a35f3f7